lib/utils/createConfig.js

```diff
@@ -136,8 +136,8 @@ function createConfig(config, argv, { port }) {
     options.https = true;
   }
 
-  if (argv.spdy) {
-    options.spdy = true;
+  if (argv.http2) {
+    options.http2 = true;
```

Comment:
Same as above

Reply:
Suggestion for a later/separate cleanup: There are a lot of `if (argv.foo) options.foo = true`. For the ones that are a simple copy:

```js
const simpleOverrides = _.pick(argv, [
  'http2',
  // other simple if argv then set options
]);

_.merge(options, simpleOverrides);
```